keras_contrib/backend/theano_backend.py

```diff
@@ -117,6 +121,24 @@ def extract_image_patches(X, ksizes, strides, border_mode="valid", dim_ordering=
     return patches
 
 
+def depth_to_space(input, scale):
```

Comment:
Cant test it on its own. That's why I am using it inside the SubPixelUpscaling layer test to test it at the same time. 

The fact that the theano implementation returns the same result as the tensorflow wrapper shows that is is correct.